nuxeo-features/nuxeo-platform-csv-export/src/main/java/org/nuxeo/ecm/platform/csv/export/computation/CSVProjectionComputation.java

```diff
@@ -125,4 +130,32 @@ protected String getData(String csv, String recordSeparator) {
         return csv.substring(csv.indexOf(recordSeparator) + recordSeparator.length());
     }
 
+    protected List<String> getList(Map<String, Serializable> params, String key) {
```

Comment:
Why do you pass in 2 params? You could just pass in `params.get(key)`

Reply:
I could, did not think about it though. Still I believe this is best like this as it's more re-usable and testable

Reply:
I didn't get your point, now I do, you are right, done.